src/events/openwhisk.js

```diff
@@ -69,7 +72,6 @@ function openWhiskWrapper(wrappedFunction) {
                     invokeEvent,
                     brief,
                     {
-                        activation_id: res.activationId,
```

Comment:
why do you remove the response activation id?

Reply:
For some reason it was duplicated - it is already sent in the "brief"